crates/forge_analyzer/src/ir.rs

```diff
@@ -63,12 +64,12 @@ pub(crate) enum Terminator {
     },
 }
 
-#[derive(Clone, Debug, Copy, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq)]
 pub enum Intrinsic {
-    Authorize,
+    Authorize(String),
```

Comment:
Should probably create an enum representing the type of API call instead of storing the raw `requestJira/requestConfluence`, since it's not clear what exactly the raw String is supposed to represent.